saleor/graphql/product/mutations/product/product_update.py

```diff
@@ -48,10 +48,9 @@ def clean_attributes(
         return attributes
 
     @classmethod
-    def post_save_action(cls, info: ResolveInfo, instance, cleaned_input):
```

Comment:
Just small note: In productCreate mutation you did leave `instance` in function definition, I think we should be consistent in naming.